webpage/src/getting-started/markdown.md

```diff
@@ -91,8 +91,9 @@ If you press <kbd>Enter</kbd> at the end of a list a new list item will be creat
 
 ## Links
 
-There are multiple ways to create links.
+Links can be created to webpages and to other notes. There are multiple ways to create links for each.
```

Comment:
Can you please write something like `Links can for example be created to webpages and to other notes.`, because there are a lots more links, like links to todo list entries, links to Nextcloud Deck cards.